src/renderers/webgl-fallback/nodes/GLSLNodeBuilder.js

```diff
@@ -476,7 +476,7 @@ ${ flowData.code }
 
 			const array = dataAttribute.array;
 
-			if ( ( array instanceof Uint32Array || array instanceof Int32Array || array instanceof Int16Array ) === false ) {
+			if ( ( array instanceof Uint32Array || array instanceof Int32Array ) === false ) {
```

Comment:
@sunag I do not fully understand why `getTypeFromAttribute()` needs to modify the data type of the attribute in the shader based on the geometry data array type. I think it would be best to generate the correct shade type right from the beginning. 

`iuvec*` types should only be generated when `attribute.gpuType === IntType` or when geometry data array is of type `Uint32Array` or `Int32Array`.

Reply:
Ok for me, I don't remember exactly when this was done, so you can proceed as you see fit.